### PR TITLE
Document trajectory execution manager

### PR DESCRIPTION
**Pull request by davetcoleman**

### CODE REVIEW

File: doc/controller_configuration/controller_configuration_tutorial.rst [hunk 1]
```diff
@@ -106,4 +106,13 @@ When you run a `move group node <../move_group_interface/move_group_interface_tu
     <remap from="joint_states" to="robot/joint_states"/>
   </node>
 
-Or you can make a subscriber with the correct topic name and then ensure that the starting robot state for your move group corresponds to a correct joints angle by using the call back of this subscriber.   
+Or you can make a subscriber with the correct topic name and then ensure that the starting robot state for your move group corresponds to a correct joints angle by using the call back of this subscriber.
+
+Trajectory Execution Manager Options
+------------------------------------
+
+There are several options for tuning the behavior and safety checks of the execution pipeline in MoveIt!. In your ``moveit_config`` package exit the ``trajectory_execution.launch.xml`` file to change the following parameters:
```
Comment (mlautman) [inline doc/controller_configuration/controller_configuration_tutorial.rst]: edit not exit

File: doc/controller_configuration/controller_configuration_tutorial.rst [hunk 2]
```diff
@@ -106,4 +106,13 @@ When you run a `move group node <../move_group_interface/move_group_interface_tu
...
+
+ - execution_duration_monitoring: when false, will not throw error is trajectory takes longer than expected to complete at the low-level controller side
```
Comment (mlautman) [inline doc/controller_configuration/controller_configuration_tutorial.rst]: I have been using double ` for code and variable names. I can make a pass to enforce consistency across the entire repo afterwords

**Comment by mlautman** [comment]
Not sure why you made changes in the davetcoleman repo but I have to reopen the pr to make these changes.

**Comment by davetcoleman** [comment]
i like to keep all my PRs branched from my personal repos... this is the standard for the moveit project

thanks for reopening this

**Pull request closed without merging**